tst/testall.g

```diff
@@ -1,3 +1,4 @@
+LoadPackage( "CaratInterface" );
```

Comment:
Ahh, of course, I should have realized this was missing. I even looked at this file. Sorry